### ISSUE TITLE
`ComposeRedundantComposable` false negative on composable lambda

### ISSUE THREAD

**Issue by JakeWharton** [slackhq/compose-lints#615]
```kotlin
val Lambda = @Composable { }
```
Nothing flagged with `com.slack.lint.compose:compose-lint-checks:1.5.4`
